src/lib/Recapitulatif.js

```diff
@@ -19,13 +20,33 @@ export const getIndividuByStep = (step, component) => {
 }
 
 export const SIMPLE_STEPS = {
+  ressources(step) {
+    const answer = getStepAnswer(this.$store.state.answers.all, step)
```

Comment:
```suggestion
    const answer = getStepAnswer(this.$store.state.answers.current, step)
```
?

Lors d'un test sur la deploy, l'étape revenus apparait même lorsqu'on a uniquement saisi et validé la date de naissance.